pkg/webhook/clusterresourceplacement/v1alpha1_clusterresourceplacement_validating_webhook.go

```diff
@@ -17,23 +16,18 @@ import (
 )
 
 const (
-	// V1Alpha1CRPValidationPath is the webhook service path which admission requests are routed to for validating ReplicaSet resources.
+	// V1Alpha1CRPValidationPath is the webhook service path which admission requests are routed to for v1alpha1 CRP resources.
 	V1Alpha1CRPValidationPath = "/validate-fleet.azure.com-v1alpha1-clusterresourceplacement"
 )
 
 type v1alpha1ClusterResourcePlacementValidator struct {
-	Client  client.Client
```

Comment:
Noticed we don't use client.Client is any of the webhook validators so removed them